.github/workflows/bundler-integrations/src/default.mjs

```diff
@@ -0,0 +1,8 @@
+import { Button } from 'govuk-frontend'
```

Comment:
**suggestion** I am wondering if it'd make sense to rename this file to make it clearer what the difference is? Either 'button' or 'single-component' rather than just 'default'?

May be more effort than it's worth at this point.